xcdat/dataset.py

```diff
@@ -1,11 +1,13 @@
 """Dataset module for functions related to an xarray.Dataset."""
 import pathlib
+from datetime import datetime
 from functools import partial
 from glob import glob
 from typing import Any, Callable, Dict, List, Literal, Optional, Tuple, Union
 
-import pandas as pd
+import cftime
 import xarray as xr
+from dateutil import relativedelta as rd
 
```

Comment:
```suggestion
from dateutil import parser

```

Reply:
This works, I just verified in another comment. Will commit this change.